Broadcast-Ext/Broadcast-Ext.xcodeproj/project.pbxproj

```diff
@@ -731,19 +765,23 @@
 				SDKROOT = iphoneos;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VALIDATE_PRODUCT = YES;
+				VALIDATE_WORKSPACE = YES;
```

Comment:
@IGitGotIt @jvalli please provide feedback. I was facing the following issue 

> /Users/gmendes/samples/opentok-ios-sdk-samples/Broadcast-Ext/Broadcast-Ext.xcodeproj: error: Building for iOS Simulator, but the linked and embedded framework 'OpenTok.framework' was built for iOS + iOS Simulator. (in target 'Broadcast-Ext' from project 'Broadcast-Ext')
/Users/gmendes/samples/opentok-ios-sdk-samples/Broadcast-Ext/Broadcast-Ext.xcodeproj: error: The linked framework 'Pods_Broadcast_Ext.framework' is missing one or more architectures required by this target: i386. (in target 'Broadcast-Ext' from project 'Broadcast-Ext')

, that was solved by allowing workspace validation. Can you see any problem by enabling this?